SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -41,6 +40,7 @@ class USpatialSender;
 class USpatialStaticComponentView;
 class USpatialWorkerConnection;
 class USpatialWorkerFlags;
+class SpatialActorGroupManager;
```

Comment:
Sort please

Reply:
Sorted, but not sure if well. Since everything here begins with U because they are UObjects, this begins with S, so goes above all of them.